### PR TITLE
Create Thai Language file

### PR DESCRIPTION
**Pull request by tomzt**
Create Thai Language file. 
Translation from English to Thai.

Hello and thank you for wanting to contribute to the MagicMirror² project

**Please make sure that you have followed these 4 rules before submitting your Pull Request:**

> 1. Base your pull requests against the `develop` branch.
>
> 2. Include these infos in the description:
>
> - Does the pull request solve a **related** issue?
> - If so, can you reference the issue like this `Fixes #<issue_number>`?
> - What does the pull request accomplish? Use a list if needed.
> - If it includes major visual changes please add screenshots.
>
> 3. Please run `npm run lint:prettier` before submitting so that
>    style issues are fixed.
>
> 4. Don't forget to add an entry about your changes to
>    the CHANGELOG.md file.

**Note**: Sometimes the development moves very fast. It is highly
recommended that you update your branch of `develop` before creating a
pull request to send us your changes. This makes everyone's lives
easier (including yours) and helps us out on the development team.

Thanks again and have a nice day!

### CODE REVIEW

File: translationsth.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	"SSW": "ต.ต.ต.ต.",
+	"SW": "ต.ต.ต.",
+	"WSW": "ต.ต.ต.ต.ต.",
+	"W": "ต.ต.",
+	"WNW": "ต.ต.ต.ต.น.",
+	"NW": "ต.ต.น.",
+	"NNW": "น.ต.ต.น.",
+
+	"FEELS": "รู้สึกเหมือน {DEGREE}",
+	"PRECIP": "ความแม่นยำ",
```
Comment (rejas) [inline translationsth.js:28]: this just got changed and split into two keys:


	"PRECIP_POP": "PoP",
	"PRECIP_AMOUNT": "Precipitation amount",

**Review by rejas — CHANGES_REQUESTED** [review]
please also rename the file to translations/th.json

**Comment by rejas** [comment]
thanks for your pullrequest. small changes necessary

please also add an entry to the CHANGELOG.md file

File: translationsth.js (hunk 1 above)
Comment (tomzt) [reply translationsth.js]: Add already

**Review by tomzt — COMMENTED** [review]
add it

**Review by rejas — APPROVED** [review]

**Review by rejas — APPROVED** [review]

**Comment by rejas** [comment]
Crap, I overlooked that it was based against master... Can we revert that on master @MichMich @khassel ?

**Comment by MichMich** [comment]
Not without creating an additional commit. :(

**Comment by MichMich** [comment]
Force push could lead to a lot of issues when others have already pulled. :(

**Comment by rejas** [comment]
Maybe we can protect the master branch from pushing PRs to it (except develop)? https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule

**Comment by MichMich** [comment]
It's already a protected branch. Maybe I'm missing something?
Screenshot 2023-02-07 at 09 57 43

**Comment by rejas** [comment]
Seems like that usecase isnt possible with github outofthebox. 
But this action could be something we add to our workflow: https://github.com/marketplace/actions/check-base-branch

**Comment by rejas** [comment]
> we should test this.

Already working on something: https://github.com/rejas/MagicMirror/pull/7

**Comment by rejas** [comment]
> Did you overlook the wrong branch when merging in the WebUI (could happen to me too)?

yes :-)

**Pull request merged**